test/tests/pbs_smoketest.py

```diff
@@ -103,7 +103,7 @@ def test_advance_reservation(self):
         jid2 = self.server.submit(j2)
 
         a = {'reserve_state': (MATCH_RE, "RESV_RUNNING|5")}
-        self.server.expect(RESV, a, id=rid, interval=1)
+        self.server.expect(RESV, a, id=rid, offset=20, interval=1)
```

Comment:
It's best to set a variable to the start time (now + 30) above.  You then set the offset here to start - time.time().  it will take into account the fact that we have slow machines in our testbed.